erpnext/education/utils.py

```diff
@@ -219,6 +220,7 @@ def get_quiz(quiz_name, course):
 	try:
 		quiz = frappe.get_doc("Quiz", quiz_name)
 		questions = quiz.get_questions()
+		duration = quiz.duration
 	except:
```

Comment:
If possible please change this statement to except the exact exception. I believe `DoesNotExistError`